schema/proto3/subnet.proto

```diff
@@ -61,9 +61,9 @@ message SubnetConfiguration {
         string mac_address = 4;
     }
 
-    repeated DnsEntry dns_list = 13;
-    repeated SecurityGroupId security_group_ids = 14;
-    repeated TransitSwitch transit_switches = 15;
+    repeated DnsEntry dns_list = 13; // FIXME: what is this for?
```

Comment:
It is a list of dns name servers associated with the subnet. Default is an empty list.

DNS servers needed to be included by dhcp.proto, right?
 

Reply:
yes, that's cover as dhcp option based on: https://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol